crates/iota/src/upgrade_compatibility.rs

```diff
@@ -33,30 +33,36 @@ use move_binary_format::{
     file_format::Visibility,
     normalized::{Enum, Function, Module, Struct},
 };
+use move_command_line_common::files::FileHash;
+use move_compiler::{
+    diag,
+    diagnostics::{
+        Diagnostic, Diagnostics,
+        codes::{DiagnosticInfo, Severity, custom},
+        report_diagnostics_to_buffer,
+    },
+    shared::files::{FileName, FilesSourceText},
+};
 use move_core_types::{
     account_address::AccountAddress,
     identifier::{IdentStr, Identifier},
     language_storage::ModuleId,
 };
+use move_ir_types::location::Loc;
 use move_package::compilation::compiled_package::CompiledUnitWithSource;
 use thiserror::Error;
 
 /// Errors that can occur during upgrade compatibility checks.
 /// one-to-one related to the underlying trait functions see:
 /// [`CompatibilityMode`]
 #[derive(Debug, Clone)]
-#[expect(dead_code)]
 pub(crate) enum UpgradeCompatibilityModeError {
     ModuleMissing {
         name: Identifier,
     },
     StructMissing {
         name: Identifier,
         old_struct: Struct,
-    },
-    StructAbilityMismatch {
-        name: Identifier,
-        old_struct: Struct,
```

Comment:
Why?

Reply:
It was an incoming change, right? I didn't do it intentionally so 🤷‍♀️

Reply:
Doesn't seem to be

Reply:
Maybe we missed a commit before this?